consul_awx_test.py

```diff
@@ -174,3 +221,13 @@ def test_get_node_meta_configfile():
         print(fp.name)
         path = fp.name
         assert get_node_meta(path) == {"k1": "v1"}
+
+
+def test_get_node_meta_types_configfile():
+    with tempfile.NamedTemporaryFile() as fp:
+        fp.write(b"[consul_node_meta_types]\ncluster:skip_bool")
+        fp.seek(0)
+        print(dir(fp))
```

Comment:
do we still need the `print`?

Reply:
I just copied from the test_get_node_meta_configfile and just changed the names.
So not sure why the print is there. :)